perun-cli/Perun/auth/OidcAuth.pm

```diff
@@ -182,6 +182,10 @@ sub authenticationRequest
 {
 	my $config = loadConfiguration();
 
+	if ($config->{"default"}) {
+		print "Environment variable PERUN_OIDC_CONFIG not set, using default config $config.\n";
```

Comment:
This would print the message even when PERUN_OIDC_CONFIG is set to the default config, which is contradictory.